presto-docs/src/main/sphinx/release/release-0.221.rst

```diff
@@ -0,0 +1,46 @@
+=============
+Release 0.221
+=============
+
+General Changes
+---------------
+* Add a variant of :func:`strpos` to return the position of the N-th instance of the substring.
+* Add :func:`strrpos` that returns position of the N-th instance of substring from the back of a string.
+* Add aggregation function :func:`entropy`.
+* Add classification aggregation functions :func:`classification_miss_rate`, :func:`classification_precision`,
+  :func:`classification_recall`, :func:`classification_thresholds`.
+* Add maximum standard error to :func:`approx_set`.
+* Fix an issue where outer joins without equality predicates would sometimes run on a single node
+  when ``join_distribution_type`` was set to ``AUTOMATIC``.
+* Add encryption for spill files (see :doc:`/admin/spill`).
+* Fix equality inference for ``VARCHAR`` columns.  This improves performance for queries that have
+  both constant predicates and join predicates on a ``VARCHAR`` column.
+* Fix a null pointer exception during the stats collecting phase in query planning.
+* Introduce ``max_tasks_per_stage`` session property and ``stage.max-tasks-per-stage`` config property.
```

Comment:
To make this consistent with others, should we do something like `Add session properties blah and blah to do something` ?